pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -0,0 +1,62 @@
+{stdenv, python, gettext, qt5, fetchFromGitHub}:
+
+stdenv.mkDerivation rec {
+  pname = "dupeguru";
+  version = "4.0.4";
+
+  src = fetchFromGitHub {
+    owner = "arsenetar";
+    repo = "dupeguru";
+    rev = "${version}";
+    sha256 = "0ma4f1c6vmpz8gi4sdy43x1ik7wh42wayvk1iq520d3i714kfcpy";
+    fetchSubmodules = true;
+  };
+
+  build_python = python.withPackages (ps: [ps.setuptools]);
+
+  nativeBuildInputs = [ 
+    gettext 
+    build_python
+    qt5.wrapQtAppsHook
+  ];
+
+  app_python = python.withPackages (ps: with ps; [
```

Comment:
The python dependencies can go to `pythonPath` attribute directly in `buildPythonApplication`.

Plus `format = "other"` to call `make` instead of `setup.py`.